backend/services/implementations/testSessionService.ts

```diff
@@ -338,6 +338,27 @@ class TestSessionService implements ITestSessionService {
     let updatedTestSession: TestSession | null;
 
     try {
+      const oldTestSession: TestSession | null = await MgTestSession.findById(
+        id,
+      );
+      if (!oldTestSession) {
+        throw new Error(`Test Session id ${id} not found`);
+      }
+
+      // If the test session is active, we don't allow teachers to update the assessment, class, or start date
+      const isActive = oldTestSession.startDate <= new Date();
```

Comment:
nit: for testability, might be good to extract a `now` parameter similar to how we've done in the past. That way we can pass in a specific `now`.